packages/common/utils/src/mapValues.ts

```diff
@@ -11,8 +13,11 @@
  * mapValues({ foo: 1, bar: 2 }, x => x * 2)
  * // => { foo: 2, bar: 4 }
  */
-export function mapValues<T, U>(value: T, mapper: (value: T[keyof T]) => U): { [K in keyof T]: U } {
-  const entries = Object.entries(value);
+export function mapValues<T extends Record<PropertyKey, T[ObjectKeys<T>]>, U>(
+  value: T,
+  mapper: (value: T[keyof T]) => U
+): { [K in keyof T]: U } {
```

Comment:
Symbol is not enumerable so it will be excluded from the result of `Object.entries`.
So you should exclude all symbol types of the result.

Reply:
OK. I understand. So I excluded symbol type in T generic. Is it right?

[fix: exclude symbol type in mapValue parameter](https://github.com/toss/slash/pull/60/commits/7f8abedca3d9d43bd3a1c0d4a5ee1f6452764efd)